tests/testsuite/run.rs

```diff
@@ -1014,6 +1019,9 @@ fn either_name_or_example() {
         .with_status(101)
         .with_stderr_data(str![[r#"
 [ERROR] `cargo run` can run at most one executable, but multiple were specified
+[HELP] available targets:
+    bin a in package foo
+    example b in package foo
```

Comment:
Do we want to quote the names? Unsure if it is more readable

```suggestion
    bin `a` in package foo
    example `b` in package foo
```

Reply:
I expect we're very inconsistent about our quoting and would be good to decide when we should and try to clean things up.